app/views/user_mailer/recipient_notification.html.erb

```diff
@@ -7,23 +7,13 @@
   </p>
 
   <p>
-    <% if @collection.nil? %>
-      <%= style_creation_link(@work.title, work_url(@work)) %> <%= "(#{@work.word_count} words)" %>
-    <% else %>
-      <%= style_creation_link(@work.title, collection_work_url(@collection, @work)) %> <%= "(#{@work.word_count} words)" %>
-    <% end %>
+    <% url = @collection ? collection_work_url(@collection, @work) : work_url(@work) %>
```

Comment:
(The work title/byline is separate from the other metadata because subscription emails display this info differently.)